### PR TITLE
docs: Added Forking and Cloning to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by adithyaanilkumar**
### Problem
Missing info about forking and cloning in the CONTRIBUTING.md file

### Solution
Added steps for Forking and Cloning the repository in CONTRIBUTING.md

### Areas of Impact
CONTRIBUTING.md

### CODE REVIEW

**Comment by MonkeyDo** [comment]
Welcome @adithyaanilkumar !
Thanks for opening your first PR with us ! :tada:

I've had a similar suggestion about explaining how to get started on git just this week, and I integrated it into an open PR to improve documentation. I wonder if this PR duplicates the effort.

I opted for a link to a [well made beginner's guide](https://akrabat.com/the-beginners-guide-to-contributing-to-a-github-project/). Please have a look at this line and let me know if you think it is sufficient: 
https://github.com/bookbrainz/bookbrainz-site/pull/320/files#diff-04c6e90faac2675aa89e2176d2eec7d8R33

Maybe instead of detailing forking and cloning in CONTRIBUTING.md, you could have a single section "New to contributing on git?" (or something to that effect), that also points to the beginner's guide?

**Comment by adithyaanilkumar** [comment]
Thank you for your suggestion. I didn't notice the effort already made by you in incorporating the beginner steps for contributing in another PR. Thank you for pointing it out to me.
The Readme.md file looks sufficient to me. 
Would you suggest that i add the same details about the guide to CONTRIBUTING.md replacing the FORKING and CLONING section that i had made during this PR,  so that it would be easier for users going straight into CONTRIBUTING.md file to see when they are trying to make a new contribution?

**Comment by MonkeyDo** [comment]
Yes, I think that would indeed be a good improvement to the docs.
As you say that means someone finding CONTRIBUTING.md will have all the information, and it can be hard to know where to look on README.md

If you want, you can add a very short step-by-step summary along with the link to the guide:

1.  fork
2. clone
3. branch
4. push branch to fork
5. open pull request

**Comment by adithyaanilkumar** [comment]
I have made the changes to the docs. I have added a short summary of everything there is to follow to start contributing to Bookbrainz. Please review the changes and please do give feedback for the same. Thank you taking time to review this.

**Comment by adithyaanilkumar** [comment]
I have made the requested  changes. Please review the pull request.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -70,3 +70,31 @@ Here's an example of a complete commit message following these guidelines:
...
+#### Commiting 
+ After adding files or making changes to an existing file,commit the changes by following the above Commiting guidelines using this command:
+  `git commit -m "Message to be typed according to the above schema"`
+
+#### Pushing branch to Fork
+  Pushing can be done by using :
+  ` git push origin BranchName`
+
+#### Pull Request
+  After pushing your changes to the remote, you can create a pull request by going to the [repository](https://github.com/bookbrainz/bookbrainz-site) page on github.do
```
Comment (MonkeyDo) [inline CONTRIBUTING.md]: There is a trailing 'do' at the end of the sentence

**Comment by adithyaanilkumar** [comment]
Hi, i have made the requested changes. Thank you for taking your time to review this PR.

**Pull request merged**